test/src/MNDiscovery/index.js

```diff
@@ -12,7 +12,7 @@ const {expect} = chai;
 
 const MockedMNList = SMNListFixture.getFirstDiff();
 
-const masternodeIps = SMNListFixture.getFirstDiff().mnList.map(masternode => masternode.service.split(':')[0]);
+const masternodeIps = SMNListFixture.getFirstDiff().mnList.map(masternode => masternode.service.substring(0, masternode.service.lastIndexOf(":")).replace("[", "").replace("]", ""));
```

Comment:
Let's make it more readable? For example, we could move `masternode.service.lastIndexOf(":")).replace("[", "").replace("]", "")` to a variable first?

Reply:
modified